passing tests for ftp upload process

alfredoliverwillder opened the pull request:
1. bundling in pry-byebug to test group
2. setting bin/console to use pry not irb
3. building Psyduck::FTPClient class
4. addding method upload_file_to_server
5. passing tests
6. renaming test --> spec convention

dmgarland commented on Gemfile:
```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 
+group :test do
```
Let's specify this as a development_dependency in the gemspec

dmgarland commented on psyduck.gemspec:
```diff
@@ -4,30 +4,35 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
   spec.version       = Psyduck::VERSION
-  spec.authors       = ["Dan Garland"]
-  spec.email         = ["dan@dangarland.co.uk"]
+  spec.authors       = ['Dan Garland']
```
Add yourself here!

Commit:
passing tests for ftp upload process

1. bundling in pry-byebug to test group
2. setting bin/console to use pry not irb
3. building Psyduck::FTPClient class
4. addding method upload_file_to_server
5. passing tests
6. renaming test --> spec convention

Commit:
moving dev dependency pry-byebug from Gemfile to gemspec

Commit:
tidying up gemspec and spec_helper
